.github/workflows/scripts/check_ignore.py

```
@@ -222,7 +222,7 @@ def main():
     print(current_branch)
     changed_files = (
         subprocess.check_output(
-            ["git", "diff", "--name-only", "develop-postgres", current_branch]
+            ["git", "diff", "--name-only", "develop", current_branch]
```

Ruff (0.11.9)

Starting a process with a partial executable path

(S607)